site/learn/expert-advice/security/ssl-vs-tls.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: advice
+title: SSL vs. TLS
+description: In this blog, you’ll learn the difference between SSL and TLS, when they are used, and why you should use them.
+author: Akira Brand
+image: advice/steps-secure-auth-system/expert-advice-ten-steps-to-secure-your-authentication-system-header-image.png
+category: Security
+date: 2021-11-12
+dateModified: 2021-11-12
+---
+
+In this blog, you’ll learn the difference between SSL and TLS, when they are used, and why you should use them.
+
+*What are SSL and TLS?*
+
+SSL (secure socket layer) and TLS (transport layer security) are a set of protocols that are used when a computer is communicating with an external server to get information.  The protocols ensure secure communication over the web by encrypting and securing any information sent over the world wide web.   
+
+*So what’s the difference between the two?*
+
+There isn’t a difference, actually.  TLS actually replaced SSL, but since SSL has been around for longer, people still refer to TLS as SSL. Confused yet?  Read on!
+
+*When do you use SSL? I mean TLS, I mean..wait, what?*
+
+You use TLS, formally known as SSL, mainly when you send data between a client and a server.
+
+*Ok, I get it. TLS used to be SSL. But how does it work?*
+
+SSL and TLS work by establishing a handshake protocol between a client and a server.  Once that handshake protocol is established, the client and server decide on a secret key that will be used to encrypt and decrypt all communications between them.
+
+In this way, an eavesdropper cannot read or manipulate any of the data sent between the client and the server, they can only see the endpoints.
+
+Now, the overall process of this handshake protocol looks like this:
+
+
+1. The client sends a “clienthello” message to the server.  This message includes information like SSL/TLS version, and the cryptographic algorithm and data compression methods that the client can support.  
+2. The server responds with a “serverhello” message. This message includes cryptographic algorithms that the server has chosen from the list provided by the client. In this way, an algorithm that both sides support is chosen. The server also sends a session Id, its digital certificate, and its public key.
+3. The client contacts the server’s certificate authority to verify the validity of the server’s certificate. This confirms the authenticity of the web server.
```

Comment:
"The client contacts the server’s certificate authority"

Is this true? I thought that it would look at the list of certificates it knows about and see if the certificate was signed by one of them.

 http://certificate.fyicenter.com/925_Trusted_Root_Certification_Authorities_in_Google_Chrome.html for example.